java/aquality-selenium-template/README.md

```diff
@@ -2,8 +2,31 @@
 
 ### Project structure
 
+- **src/test/java/aquality/selenium/template/**
+  - **configuration/**: classes that used to fetch project config from [src/test/resources/environment](./src/test/resources/environment) folder
+  - **models/**: classes that represent data models of the application under the test (POJO classes) 
+  - **forms/**: Page Objects
+  - **glue/**
```

Comment:
Do we plan to create a non-cucumber template?
If we don't, isn't it better to put all cucumber-related features to a cucumber package?

Reply:
Is it necessary? I mean non-cucumber.

Reply:
I vote for yes, don't think you will always need to use cucumber

Reply:
In this case it also makes sense to create templates for different test runners and build tools?;)

Reply:
I thought we are going to do it )